### DISCUSSION TITLE
New to MFEM and I am looking for an FE library I know I can invest energy learning.

### DISCUSSION THREAD

**Discussion by AbbBallout** [mfem/mfem#3733] [Q&A] [answered]
The tutorial examples explain problem applications on a surface level but I want to know if MFEM is customizable enough to apply my own FE space if I needed to.  Is there support for different physics in different domains? What if I want to implement something, is there a support group?

**Comment by tzanio** [comment, marked as answer]
Hi @AbbBallout,

MFEM is designed to be general and customizable, but like with every tool, there will be some learning curve. You can ask questions and get help here (through Issues or Discussions, we have 600+ members in the GitHub organizations). There is some support for different physics in different domains, but the default coupling is basic interpolation (cc: @jandrej).

Hope this helps,
Tzanio
